jekyll/_cci2_ja/browser-testing.md

```diff
@@ -92,15 +92,15 @@ jobs:
 ## Sauce Labs
 {: #sauce-labs }
 
-Sauce Labs has an extensive network of operating system and browser combinations you can test your web application against. Sauce Labs supports automated web app testing using Selenium WebDriver scripts as well as through `saucectl`, their test orchestrator CLI, which can be used to execute tests directly from a variety of JavaScript frameworks.
+Sause Labs には、Web アプリケーションをテストできるオペレーティングシステムとブラウザの組み合わせが幅広いネットワークで用意されています。 Sauce Labs は、Selenium WebDriver スクリプトを使用した自動 Web アプリテストや、さまざまな JavaScript フレームワークから直接テストを実行するためのテストオーケストレーターである CLI をサポートしています。
```

Comment:
I believe the following is not `Sause` but `Sauce`

```suggestion
Sauce Labs には、Web アプリケーションをテストできるオペレーティングシステムとブラウザの組み合わせが幅広いネットワークで用意されています。 Sauce Labs は、Selenium WebDriver スクリプトを使用した自動 Web アプリテストや、さまざまな JavaScript フレームワークから直接テストを実行するためのテストオーケストレーターである CLI をサポートしています。
```
https://circleci.com/docs/2.0/browser-testing#sauce-labs

<img width="785" alt="Screen Shot 2022-06-23 at 8 29 31" src="https://user-images.githubusercontent.com/6015450/175175581-08e13838-6eba-4fa9-99b7-58a8fc10eda2.png">